content/en/platform/corda/5.2/developing-applications/cordapp-template/reset.md

```diff
@@ -1,40 +1,41 @@
 ---
 date: '2023-11-01'
-title: "Resetting the CSDE"
+title: "Resetting the Corda Runtime Gradle Plugin"
+description: Learn stop and clean the Corda runtime Gradle plugin.
 menu:
-  corda5-tools:
-    parent: corda5-develop-get-started
-    identifier: corda5-reset
+  corda52:
+    parent: corda52-develop-get-started
+    identifier: corda52-reset
     weight: 8000
-section_menu: corda5-tools
+
 ---
-# Resetting the CSDE
+# Resetting the Corda Runtime Gradle Plugin
```

Comment:
A better title would be "Resetting the CorDapp Template environment"

Reply:
This was edited previously based on comment from @davidcurrie